Roles/Core/CustomRoleManager.cs

```diff
@@ -243,6 +252,15 @@ public static bool OnCheckMurder(ref PlayerControl killer, ref PlayerControl tar
                     DollMaster.CheckMurderAsPossessed(killer, target);
                     return false;
                 }
+
+        if (canceled)
+            return false;
+
+        if (cancelbutkill)
+        {
+            killer.RpcMurderPlayer(target);
```

Comment:
But Swift will kill twice and teleport to the dead body

Reply:
oh right, I forgot to change inside swift code 😅 fixed